classes/acrn-sblimage.bbclass

```diff
@@ -62,8 +62,9 @@ python do_acrn_sblimage() {
 python() {
     import re
 
-    multibootPackageDependencyList = re.split(" +", d.getVar('MB_DEPENDENCY').lstrip().rstrip())
-    multibootPackageMcDependencyList = re.split(" +", d.getVar('MB_MC_DEPENDENCY').lstrip().rstrip())
+   currentMultibootConfig = d.getVar('BB_CURRENT_MC')
```

Comment:
Why it is required ? Can you explain with some scenario ?
Please check MB_MC_DEPENDENCY variable..
https://github.com/intel/meta-acrn/blob/master-next/docs/slimbootloader.md#optional-other-variables

Reply:
We used as below in the conf/local.conf for vm0 and vm1 multiboot config.  THe scenario is ACRN logical partition mode.
BBMULTICONFIG = "sos vm0 vm1"

and used the 'd.getVar('BB_CURRENT_MC')' to get the current configuration.
wihtout the changes in the 'acrn-sblimage.bbclass' the build is failing while creating the Container.

Let us know if any other way to handle our scenario.

Reply:
Patch looks Ok. Can you please document BB_CURRENT_MC  variable too at https://github.com/intel/meta-acrn/blob/master-next/docs/slimbootloader.md#optional-other-variables !

Reply:
Also set BB_CURRENT_MC default to empty string.
BB_CURRENT_MC  ?= ""

Reply:
updated the slimbootloader.md and assigned the default string to the BB_CURRENT_MC variable.